src/views/ModelPlan/TaskList/ITSolutions/SolutionImplementation/index.tsx

```diff
@@ -189,32 +197,53 @@ const SolutionImplementation = () => {
     return <NotFound />;
   }
 
+  const renderCancelCopy = () => {
+    if (isUpdatingStatus && fromSolutionDetails) {
+      return t('dontUpdateandReturnToSolutionDetails');
+    }
+    if (isUpdatingStatus) {
+      return t('dontUpdateandReturnToTracker');
+    }
+    return t('dontAdd');
+  };
+
+  const handleCancelClick = (values: GetOperationalNeedOperationalNeedType) => {
+    if (isUpdatingStatus && fromSolutionDetails) {
+      return history.goBack();
+    }
+    if (isUpdatingStatus) {
+      return history.push(`/models/${modelID}/task-list/it-solutions`);
+    }
+    return handleFormSubmit(values, null, true);
+  };
+
+  const breadcrumbs = [
+    { text: h('home'), url: '/' },
+    { text: h('tasklistBreadcrumb'), url: `/models/${modelID}/task-list/` },
+    { text: t('breadcrumb'), url: `/models/${modelID}/task-list/it-solutions` },
+    {
+      text: t('solutionDetails'),
+      url: `/models/${modelID}/task-list/it-solutions/${operationalNeed.id}/${operationalNeed.solutions[0]?.id}/solution-details`
+    },
+    { text: isUpdatingStatus ? t('updateStatus') : t('selectSolution') }
+  ];
+
+  const solutionsArray =
+    solutionId === undefined
+      ? operationalNeed.solutions
+      : operationalNeed.solutions.filter(
+          solution => solution.id === solutionId
+        );
+
```

Comment:
This filtering on the solutions need to be passed to the Formik context.  All the solutions are still being passed to to Formik and attempting to update all solutions on the mutation.  `operationalNeed` is being passed, along with nested solutions on line 285.  The filtering needs to happen on this variable.  

```
const formikNeed = {...operationalNeed}
  if (operationalNeedID) {
    formikNeed.solutions = formikNeed.solutions.filter(solution => solution.id === solutionId)
  }

or

formikNeed.solutions = solutionsArray;
```